lib/functions.php

```diff
@@ -414,7 +414,7 @@ function elgg_solr_get_adapter_options() {
 function elgg_solr_has_settings() {
 	$host = elgg_get_plugin_setting('host', 'elgg_solr');
 	$port = elgg_get_plugin_setting('port', 'elgg_solr');
-	$path = elgg_get_plugin_setting('path', 'elgg_solr');
+	$path = elgg_get_plugin_setting('solr_path', 'elgg_solr');
```

Comment:
ok, I will merge this sometime over the weekend.  I've been trying to keep it compatible back to 1.8 as far as I could but now seems like a good a time as any to make the break.  There's a few other things I need to do with it so will merge when I have more time to really check it out.